protocol/contracts/src/reserve/ReserveComptroller.sol

```diff
@@ -210,4 +253,9 @@ contract ReserveComptroller is ReserveAccessors, ReserveVault {
     function _fromUsdcAmount(uint256 usdcAmount) internal pure returns (uint256) {
         return usdcAmount.mul(USDC_DECIMAL_DIFF);
     }
+
+    function _canBorrow(address account) private pure returns (bool) {
```

Comment:
Do we want to have a total or per address debt maximum?

Reply:
Great idea 👍

Reply:
Great idea 👍